deploy/New-IdentityProvider.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+.Description
+This allows you to create an Identity Provider for providing tokens
+to an application - this is optional.
+#> 
+
+[CmdletBinding()]
+param(
+    # The name of the AAD Client Application
+    [Parameter(Mandatory=$false)]
+    [string] $AADClientAppName="ccf_demo_client",
+    # The name of the AAD API Application
+    [Parameter(Mandatory=$false)]
+    [string] $AADApiAppName="ccf_demo_api",
+    # Tenant where the Application Registrations are
+    [Parameter(Mandatory=$true)]
+    [string] $TenantId
+)
+
+# The Magic Strings are well known Ids
+$MSGraphAppId="00000003-0000-0000-c000-000000000000"
+# $Scope_OpenID="37f7f235-527c-4136-accd-4a02d197296e"
+# $Scope_OfflineAccess="7427e0e9-2fba-42fe-b0c0-848c9e6a8182"
+$Scope_UserRead="e1fe6dd8-ba31-4d61-89e7-88639da4683d"
+
+# Hold all of our keys in a hashtable
+$output = @{}
+# Clean our .env file
+Remove-Item ../.env -Force -ErrorAction SilentlyContinue
```

Comment:
Could this fail if there is no `.env` before?

Reply:
Has a SilentlyContinue on the end so will ignore any issues.